tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/types/string/StringTemplateLiteralTest.java

```diff
@@ -237,4 +237,13 @@ public void concatStringTemplateExprs() {
         Assert.assertTrue(returns[0] instanceof BString);
         Assert.assertEquals(returns[0].stringValue(), "FirstName: John. Second name: Doe");
     }
+
+    @Test
+    public void testComplexStringTemplateExpr() {
+        BValue[] args = {};
+        BValue[] returns = BRunUtil.invoke(result, "complexStringTemplateExpr", args);
```

Comment:
```suggestion
        BValue[] returns = BRunUtil.invoke(result, "complexStringTemplateExpr", new BValue[0]);
```

Reply:
@NatashaWso2: Followed the same convention used in the other test cases of `StringTemplateLiteralTest.java`. Other suggestions addressed with commit 692c79d.